test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -145,6 +145,66 @@ Describe "Start-Transcript, Stop-Transcript tests" -tags "CI" {
         Test-Path $transcriptFilePath | Should -BeTrue
 
         $machineName = [System.Environment]::MachineName
-        $transcriptFilePath | Should FileContentMatch $machineName
+        $transcriptFilePath | Should -FileContentMatch $machineName
+    }
+
+    It "Transcription should record Write-Information output when InformationAction is set to Continue" {
+        [String]$message = New-Guid
+        $script = {
+            Start-Transcript -Path $transcriptFilePath
+            Write-Information -Message $message -InformationAction Continue
+            Stop-Transcript }
```

Comment:
Please use common formatting.
```powershell
$script = {  
    Start-Transcript -Path $transcriptFilePath  
    Write-Information -Message $message -InformationAction Continue  
    Stop-Transcript
}
```

Reply:
I see. I believe we should use the right pattern because It block and other code use right patterns.

Reply:
Fixed

Reply:
Seems it is an isolation. I'am not sure that we should remove thos.